### ISSUE TITLE
Regarding the evaluation on multi-instance-retrieval

### ISSUE THREAD

**Issue by forrestsz** [epic-kitchens/C5-Multi-Instance-Retrieval#1]
Dear EPIC-KITCHENS organizers, 

Thank you for your contributions for the promising epic-kitchens benchmark, I am very interested in your multi-instance retrieval task. 

I have read your paper and code from MI-MM, and found that evaluation benchmark uses **mAP** and **nDCG** metrics, which is different from the popular used **Recall**. nDCG requires **an relevancy matrix** as input, commented as `EPIC_100_retrieval_{}_relevancy_mat.pkl` in your code.

Currently, only the relevancy matrix on training set is provided, with shape `(67217, 15990)`, and test set matrix is not provided, and not val set.

So, If i want to evaluate the model off-line before submission, should I sample a portion of the training set as a validation set to compare performance?

Many thanks for your help, I appreciate your response.

**Comment by mwray** [comment]
Hi,

Thanks for your interest in the benchmark! Yes, if you want to evaluate offline you will need to sample a portion of the training set to be a validation set. We cannot release the relevancy matrix of the test set as that would leak information about the challenge.

Thanks,

Michael
